javascript/operations.js

```diff
@@ -173,7 +179,7 @@ export default {
       before(element, operation)
       operate(operation, () => {
         const { name } = operation
-        element.classList.add(...getClassNames(name || ''))
+        element.classList.add(...getClassNames(name != null ? name : ''))
```

Comment:
It appears that `getClassNames` is not returning what's expected. 

<img width="372" alt="Search · getClassNames 2022-07-11 08-01-01" src="https://user-images.githubusercontent.com/32920/178281721-00c990f2-98ef-4583-bcad-569bf83c544c.png">

We may want to update it to something like this.

```js
const getClassNames = names => Array.from(names).flat()
```

Then update all usages to pass an Array argument on invocation.

```js
element.classList.add(...getClassNames([name != null ? name : '']))
```